data/ms/legislature/Earle-S-Banks-4fa41145-35bf-442b-bdfe-cf54ea53da69.yml

```diff
@@ -2,30 +2,35 @@ id: ocd-person/4fa41145-35bf-442b-bdfe-cf54ea53da69
 name: Earle S. Banks
 given_name: Earle S.
 family_name: Banks
-email: ebankslaw@aol.com
+email: ebankslaw@aol.com@house.ms.gov
```

Comment:
@NewAgeAirbender This email looks wrong but I'm not sure whether it should be ebankslaw@aol.com or ebankslaw@house.ms.gov

Reply:
@maxknutsen it should be ebankslaw@aol.com thats whats listed on the MS website on his webpage